benchmarks/ner_embeddings/scripts/symbol_stats.py

```diff
@@ -0,0 +1,130 @@
+import tqdm
```

Comment:
We did not use this script in the end to report collision statistics in the paper. Should we just not have this script? Maybe its confusing?

Reply:
Sure can remove